src/main.rs

```diff
@@ -634,6 +655,8 @@ fn start_vmm(cmd_arguments: ArgMatches) -> Result<Option<String>, Error> {
     let vm_debug_evt = EventFd::new(EFD_NONBLOCK).map_err(Error::CreateDebugEventFd)?;
 
     let exit_evt = EventFd::new(EFD_NONBLOCK).map_err(Error::CreateExitEventFd)?;
+    // TODO: landlock_enable is being parsed in 2 places: here and VMparams:parse() how to merge both?
```

Comment:
I rechecked and this comment is not entirely true. I am only reading from parsed results(ArgMatches), instead of re-parsing the cmdline options. 

I see similar uses in other places, so I dropped the comment.